fix_: "to" chains being used in sending bridge tx via hop are more explicit

saledjenic opened the pull request:
Using `ChainIDTo` depicts better an intention which chain should be used.

saledjenic committed:
fix_: "to" chains being used in sending bridge tx via hop are more explicit

Using `ChainIDTo` depicts better an intention which chain should be used.

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (10)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | e3c8e96b | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5574/1/) | 2024-07-24 21:57:36 | ~2 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5574/1//consoleText) |
| :heavy_check_mark: | e3c8e96b | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5574/1/) | 2024-07-24 21:59:11 | ~4 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240724-215508-e3c8e9-pr5574.zip) |
| :heavy_check_mark: | e3c8e96b | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5574/1/) | 2024-07-24 21:59:43 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240724-215503-e3c8e9-pr5574.zip) |
| :heavy_check_mark: | e3c8e96b | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5574/1/) | 2024-07-24 22:00:31 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240724-215503-e3c8e9-pr5574.aar) |
| :heavy_multiplication_x: | e3c8e96b | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5574/1/) | 2024-07-24 22:40:51 | ~45 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5574/1//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 95b61a18 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5574/2/) | 2024-07-25 07:17:23 | ~2 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5574/2//consoleText) |
| :heavy_check_mark: | 95b61a18 | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5574/2/) | 2024-07-25 07:18:00 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240725-071443-95b61a-pr5574.zip) |
| :heavy_check_mark: | 95b61a18 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5574/2/) | 2024-07-25 07:18:49 | ~4 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240725-071443-95b61a-pr5574.zip) |
| :heavy_check_mark: | 95b61a18 | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5574/2/) | 2024-07-25 07:20:11 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240725-071443-95b61a-pr5574.aar) |
| :heavy_check_mark: | 95b61a18 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5574/2/) | 2024-07-25 08:01:01 | ~46 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5574/2//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 51989119 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5574/3/) | 2024-07-25 07:32:38 | ~2 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5574/3//consoleText) |
| :heavy_check_mark: | 51989119 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5574/3/) | 2024-07-25 07:32:54 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240725-073027-519891-pr5574.zip) |
| :heavy_check_mark: | 51989119 | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5574/3/) | 2024-07-25 07:34:15 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240725-073027-519891-pr5574.zip) |
| :heavy_check_mark: | 51989119 | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5574/3/) | 2024-07-25 07:35:39 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240725-073023-519891-pr5574.aar) |
| :heavy_multiplication_x: | 51989119 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5574/3/) | 2024-07-25 08:41:32 | ~40 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5574/3//consoleText) |
| | | | | | | |
| :heavy_check_mark: | a75c0cd9 | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5574/4/) | 2024-07-25 10:44:05 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240725-104154-a75c0c-pr5574.aar) |
| :heavy_check_mark: | a75c0cd9 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5574/4/) | 2024-07-25 10:44:14 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240725-104156-a75c0c-pr5574.zip) |
| :heavy_check_mark: | a75c0cd9 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5574/4/) | 2024-07-25 10:44:28 | ~2 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5574/4//consoleText) |
| :heavy_check_mark: | a75c0cd9 | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5574/4/) | 2024-07-25 10:45:48 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240725-104155-a75c0c-pr5574.zip) |
| :heavy_multiplication_x: | a75c0cd9 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5574/4/) | 2024-07-25 11:24:13 | ~42 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5574/4//consoleText) |

saledjenic committed:
chore_: checking for required balance improved

An error contains now more details, for which token on which chain there is not enough balance for
executing a tx. Also check for required balance now calculates in fees for all tx that possibly can be
sent from the same chain.
